src/api/app/assets/javascripts/webui2/tabs.js

```diff
@@ -27,7 +27,7 @@ function refreshTabs(tabList) {
 
 $.fn.hasOverflow = function() {
   var element = $(this)[0];
-  return (element.offsetHeight < element.scrollHeight) || (element.offsetWidth < element.scrollWidth);
+  return (element.offsetHeight < element.scrollHeight - 1) || (element.offsetWidth < element.scrollWidth - 1);
```

Comment:
you heard of <= ? :)

Reply:
Obviously never, right? :man_facepalming:

Reply:
It's not the same though.

Because with `<=`, it will keep being true when the scroll and offset are the same, so this will shrink all the tabs even though it should not.

Reply:
OK, let's agree on: this needs a little comment :)